index.html

```diff
@@ -178,36 +178,36 @@ <h1>Uber Navigation</h1>
                     <div class="connect">LET'S CONNECT</div>
                 </p>
                 <ul>
-                    <li><a href=""><img src="./images/github.png" /></a></li>
-                    <li><a href=""><img src="./images/linkedin.jpeg" /></a></li>
-                    <li><a href=""><img src="./images/sign.png" /></a></li>
-                    <li><a href=""><img src="./images/twiter.png" /></a></li>
-                    <li><a href=""> <img src="./images/Muliti.png" /></a></li>
+                    <li><a href=""><img src="./images/github.png" alt="here is the link to github"/></a></li>
+                    <li><a href=""><img src="./images/linkedin.jpeg" alt="here is the link to linkedin" /></a></li>
+                    <li><a href=""><img src="./images/sign.png" alt="here is the link to angellist" /></a></li>
+                    <li><a href=""><img src="./images/twiter.png" alt="here is the link to twitter" /></a></li>
+                    <li><a href=""> <img src="./images/Muliti.png" alt="here is the link to Muliti"/></a></li>
                 </ul>
 
                 <div class="get">Get my resume</div>
```

Comment:
- [ ] Kindly as the previous reviewer asked, you have low contrast at this div `<div class="get">Get my resume</div>` so you just need to change the color of the item to a darker one to pass. you can use `#865AE5` for the div text.

![image](https://user-images.githubusercontent.com/35535130/156378940-91c2e875-245b-4f7e-9c45-a8a0d423e92d.png)